store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/integration/FederatedViewsIT.java

```diff
@@ -180,39 +177,41 @@ public void shouldNotAddAndGetEdgeWithEntityGraph() throws OperationException {
                                 .build())
-                        .option(FederatedStoreConstants.KEY_OPERATION_OPTIONS_GRAPH_IDS, ACCUMULO_GRAPH_WITH_ENTITIES)
-                        .build(), user))
-                .withMessage("Operation chain is invalid. Validation errors: \n"
-                        + "View is not valid for graphIds:[AccumuloStoreContainingEntities]\n"
-                        + "(graphId: AccumuloStoreContainingEntities) View for operation uk.gov.gchq.gaffer.operation.impl.get.GetAllElements is not valid. \n"
-                        + "(graphId: AccumuloStoreContainingEntities) Edge group BasicEdge does not exist in the schema");
+                        .build())
+                        .graphIdsCSV(GRAPH_ID_ACCUMULO_WITH_ENTITIES), user))
+                .withMessage("Operation chain is invalid. Validation errors: \n" +
+                        "View is not valid for graphIds:[" + GRAPH_ID_ACCUMULO_WITH_ENTITIES + "]\n" +
+                        "(graphId: " + GRAPH_ID_ACCUMULO_WITH_ENTITIES + ") View for operation uk.gov.gchq.gaffer.federatedstore.operation.FederatedOperation is not valid. \n" +
+                        "(graphId: " + GRAPH_ID_ACCUMULO_WITH_ENTITIES + ") Edge group BasicEdge does not exist in the schema");
     }
 
+
     /**
      * Federation acts as a Edge graph with a view of Entity
      *
      * @throws OperationException any
      */
     @Test
-    public void shouldNotAddAndGetEntityWithEntityGraph() throws OperationException {
+    public void shouldNotAddAndGetEntityWithEdgeGraph() throws OperationException {
 
         addBasicEntity();
 
         assertThatExceptionOfType(Exception.class)
-                .isThrownBy(() -> graph.execute(new GetAllElements.Builder()
-                        .view(new View.Builder()
-                                .entity(BASIC_ENTITY)
+                .isThrownBy(() -> graph.execute(FederatedStoreUtil.<GetAllElements>getFederatedOperation(new GetAllElements.Builder()
+                                .view(new View.Builder()
+                                        .entity(GROUP_BASIC_ENTITY)
+                                        .build())
                                 .build())
-                        .option(FederatedStoreConstants.KEY_OPERATION_OPTIONS_GRAPH_IDS, ACCUMULO_GRAPH_WITH_EDGES)
-                        .build(), user))
-                .withMessage("Operation chain is invalid. Validation errors: \n"
-                        + "View is not valid for graphIds:[AccumuloStoreContainingEdges]\n"
-                        + "(graphId: AccumuloStoreContainingEdges) View for operation uk.gov.gchq.gaffer.operation.impl.get.GetAllElements is not valid. \n"
-                        + "(graphId: AccumuloStoreContainingEdges) Entity group BasicEntity does not exist in the schema");
+                        .graphIdsCSV(GRAPH_ID_ACCUMULO_WITH_EDGES), user))
+
+                .withMessage("Operation chain is invalid. Validation errors: \n" +
```

Comment:
Can we either remove the \n, or mark this class as @DisabledOnOs(OS.WINDOWS) as we have done with other tests.